platform/windows/export/export_plugin.cpp

```diff
@@ -513,9 +497,24 @@ Error EditorExportPlatformWindows::_rcedit_add_data(const Ref<EditorExportPreset
 	}
 
 #ifndef WINDOWS_ENABLED
-	// On non-Windows we need WINE to run rcedit
-	args.push_front(rcedit_path);
-	rcedit_path = wine_path;
+	// On non-Windows we need WINE to run rcedit unless its being run via PATH
+	if (rcedit_path != "rcedit") {
+		String wine_path = EDITOR_GET("export/windows/wine");
+
+		if (!wine_path.is_empty() && !FileAccess::exists(wine_path)) {
+			add_message(EXPORT_MESSAGE_WARNING, TTR("Resources Modification"), vformat(TTR("Could not find wine executable at \"%s\"."), wine_path));
+			return ERR_FILE_NOT_FOUND;
+		}
+
+		if (wine_path.is_empty()) {
+			wine_path = "wine"; // try to run wine from PATH
+		}
+
+		args.push_front(rcedit_path);
+		rcedit_path = wine_path;
+		setenv("WINEDEBUG", "-all", true); // Disable wine debug messages
+		setenv("WINEDLLOVERRIDES", "mscoree=", true); // Prevent wine-mono install prompt on first run
```

Comment:
I suggest using `OS::get_singleton()->set_environment()` instead for consistency with the rest of the engine, even if this block is UNIX-specific.

Reply:
Just changed it, thanks.